src/components/layout/sendTip/SendPost.vue

```diff
@@ -32,15 +32,17 @@
         <ButtonPlain @click="showGifs = !showGifs">
           <IconGif :class="{ active: showGifs }" />
         </ButtonPlain>
-        <ButtonPlain disabled>
-          <IconEmoji />
-        </ButtonPlain>
-        <ButtonPlain disabled>
-          <IconPoll />
-        </ButtonPlain>
-        <ButtonPlain disabled>
-          <IconThreeDots />
-        </ButtonPlain>
+        <template v-if="UNFINISHED_FEATURES">
```

Comment:
Proposing to mark this as unfinished instead of commenting to keep code highlighting, simplify refactoring of related parts and so on.
We have been using this approach before in Base aepp: https://github.com/aeternity/aepp-base/blob/develop/.env.production#L1